smart_control/reinforcement_learning/utils/config.py

```diff
@@ -35,86 +34,142 @@
+    "starter_buffers",
+)
+EXPERIMENT_RESULTS_PATH = os.path.join(
+    ROOT_DIR, "smart_control", "reinforcement_learning", "experiment_results"
+)
 
 
 @gin.configurable
 def get_histogram_path() -> str:
-    """Get path to histogram data.
-    
-    Returns:
-        Path to histogram data.
-    """
-    return DATA_PATH
+  """Get path to histogram data.
+
+  Returns:
+      Path to histogram data.
+  """
+  return DATA_PATH
 
 
 @gin.configurable
 def get_reset_temp_values() -> np.ndarray:
-    """Get reset temperature values.
-    
-    Returns:
-        Reset temperature values.
-    """
-    reset_temps_filepath = os.path.join(DATA_PATH, "reset_temps.npy")
-    
-    return np.load(reset_temps_filepath)
+  """Get reset temperature values.
+
+  Returns:
+      Reset temperature values.
+  """
+  reset_temps_filepath = os.path.join(DATA_PATH, "reset_temps.npy")
+
+  return np.load(reset_temps_filepath)
 
 
 @gin.configurable
 def get_zone_path() -> str:
-    """Get path to zone data.
-    
-    Returns:
-        Path to zone data.
-    """
-    return os.path.join(DATA_PATH, "double_resolution_zone_1_2.npy")
+  """Get path to zone data.
+
+  Returns:
+      Path to zone data.
+  """
+  return os.path.join(DATA_PATH, "double_resolution_zone_1_2.npy")
 
 
 @gin.configurable
 def get_metrics_path() -> str:
-    """Get path to metrics.
-    
-    Returns:
-        Path to metrics.
-    """
-    return os.path.join(METRICS_PATH, "metrics")
+  """Get path to metrics.
+
+  Returns:
+      Path to metrics.
+  """
+  return os.path.join(METRICS_PATH, "metrics")
 
 
 @gin.configurable
 def get_weather_path() -> str:
-    """Get path to weather data.
-    
-    Returns:
-        Path to weather data.
-    """
-    return os.path.join(DATA_PATH, "local_weather_moffett_field_20230701_20231122.csv")
+  """Get path to weather data.
+
+  Returns:
+      Path to weather data.
+  """
+  return os.path.join(
+      DATA_PATH, "local_weather_moffett_field_20230701_20231122.csv"
+  )
 
 
 @gin.configurable
 def get_histogram_reducer() -> Any:
-    """Get histogram reducer.
-    
-    Returns:
-        Histogram reducer.
-    """
-    
-    histogram_parameters_tuples = (
-        ('zone_air_temperature_sensor', (285., 286., 287., 288, 289., 290., 291.,
-        292., 293., 294., 295., 296., 297., 298., 299., 300., 301, 302, 303)),
-        ('supply_air_damper_percentage_command', (0.0, 0.2, 0.4, 0.6, 0.8, 1.0)),
-        ('supply_air_flowrate_setpoint', (0., 0.05, .1, .2, .3, .4, .5,  .7,  .9)),
-    )
-    
-    reader = controller_reader.ProtoReader(DATA_PATH)
-    
-    hr = histogram_reducer.HistogramReducer(
-        histogram_parameters_tuples=histogram_parameters_tuples,
-        reader=reader,
-        normalize_reduce=True,
-    )
-    return hr
+  """Get histogram reducer.
+
+  Returns:
+      Histogram reducer.
+  """
+
+  histogram_parameters_tuples = (
+      (
+          "zone_air_temperature_sensor",
+          (
+              285.0,
```

Comment:
echo comments from below about reverting this number-per-line formatting behavior